email_composer/src/components/DropfileField.js

```diff
@@ -12,9 +12,9 @@ const DropfileField = props => (
       (props.isDragActive ? ' dragActive' : '') +
       (props.isToolbarHidden ? ' toolbarHidden' : '')
     }
-    onDragLeave={props.onDragLeave}
-    onDragOver={props.onDragOver}
-    onDrop={props.onDrop}
+    onDragLeave={e => props.onDragLeave(e)}
+    onDragOver={e => props.onDragOver(e)}
+    onDrop={e => props.onDrop(e)}
```

Comment:
why not just `onDrop={props.onDrop}` ?

Reply:
that know that this is the end spread of this props

Reply:
I can live with that